suricata/update/rule.py

```diff
@@ -146,6 +146,8 @@ def __str__(self):
         return self.format()
 
     def format(self):
+        if self.noalert and not "noalert;" in self.raw:
+            self.raw = re.sub(r'( *sid\: *[0-9]+\;)', r' flowbits:noalert;\1', self.raw)
```

Comment:
I forgot to say: you can simply use 'noalert'. There is no need to have the flowbits: prefix.

Reply:
I agree, new pull request is open: https://github.com/OISF/suricata-update/pull/141